app/assets/stylesheets/overrides.scss

```diff
@@ -74,16 +126,44 @@ $font: $brown;
   padding: 15px;
 }
 
-table.center {
-  margin-left: auto;
-  margin-right: auto;
-  max-width: 500px;
-}
-
 .has-table {
   padding-bottom: 20px;
 }
 
 .example-table {
   font-style: italic;
+  align: center;
+}
+
+.exercise-table {
+  align: center;
+}
+
+.main-title a {
+  color: $titlecolor;
+}
+
+.button.round {
+  background-color: $navborder;
+  color: $navmainfont;
+}
+
+.full-width {
+  width: 100%;
+}
+
+#notice_wrapper {
```

Comment:
Avoid using id selectors<br>Selector `notice_wrapper` should be written in lowercase with hyphens